.github/workflows/upload.yml

```
@@ -29,7 +31,7 @@ jobs:
           mv mrmustard src
 
       - name: Install only test dependencies
-        run: uv sync --inexact --no-install-project --all-extras
+        run: uv sync --only-dev --inexact --no-install-project
 
       - name: Run tests
         run: uv run --no-project pytest tests -p no:warnings --tb=native --backend=tensorflow
```

I think this `uv run` should still be avoided?
```suggestion
        run: pytest tests -p no:warnings --tb=native --backend=tensorflow
```